### PR TITLE
Adding conference header

### PR DESCRIPTION
**Pull request by AndrewBestbier**

### CODE REVIEW

File: app/components/Conference/index.js [hunk 1]
```diff
@@ -1,111 +1,72 @@
...
-            </form>
+  <div>
+    <div id="wrapper">
+      <main id="conference">
+        <Hero page="Conference" />
+        <RedBadgerBanner />
+        <nav className="NavigationBar block">
+          <ul className="NavigationBar__tabs">
+            <li><a href="/">Meetups</a></li>
+            <li><a className="NavigationBar__tab--active" href="#">Conference</a></li>
```
Comment (lpil) [inline app/components/Conference/index.js]: href required?

Comment (AndrewBestbier) [inline app/components/Conference/index.js]: Actually didn't change that - was there previously. After this Hero pull is merged I want to build a Navbar component - can fix it there?

**Comment by lpil** [comment]
LGTM

**Comment by AndrewBestbier** [comment]
Alt Text

File: app/components/Hero/index.js [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
+        src={`/img/PNG/${page}_header_transparent.png`}
         alt="Red Badger logo"
       />
     </object>
     <h1 className="Hero__title">React London</h1>
   </header>
 );
 
+Hero.propTypes = {
+  page: PropTypes.oneOf(['Conference', 'Community']).isRequired,
```
Comment (zoeabryant) [inline app/components/Hero/index.js]: 👍 niiiiice

File: app/components/Hero/index.scss [hunk 3]
```diff
@@ -13,7 +13,7 @@
   flex-direction: column;
   align-content: center;
 
-  z-index: -1;
+  z-index: 999;
```
Comment (zoeabryant) [inline app/components/Hero/index.scss]: see https://css-tricks.com/handling-z-index/

**Comment by zoeabryant** [comment]
👍  I'm happy

**Pull request merged**